core/src/config.jl

```diff
@@ -120,6 +120,7 @@ end
 @option @addnodetypes struct Toml <: TableOption
     starttime::DateTime
     endtime::DateTime
+    crs::String
```

Comment:
Julia doesn't use this, so why not give it a default? That would help with older toml files as well.